### PR TITLE
std: small doc fixes for BufReader and BufWriter

### PR DESCRIPTION
**Pull request by birkenfeld**
- fix probable copy-paste error in BufWriter.get_mut()
- more consistent punctuation

### CODE REVIEW

File: src/libstd/io/buffered.rs [hunk 1]
```diff
@@ -21,7 +21,7 @@ use io::{self, DEFAULT_BUF_SIZE, Error, ErrorKind, SeekFrom};
 use ptr;
 use iter;
 
-/// Wraps a `Read` and buffers input from it
+/// Wraps a `Read` and buffers input from it.
```
Comment (steveklabnik) [inline src/libstd/io/buffered.rs]: Convention is to use full sentences, actually, so I'd prefer to leave this (and others)

**Comment by steveklabnik** [comment]
Thanks @birkenfeld ! I'll take the typo fixes, but the periods should stay.

**Comment by birkenfeld** [comment]
I don't understand - I'm *adding* the periods, which are also present in the first sentences of other docstrings in this file.

**Comment by steveklabnik** [comment]
@birkenfeld sorry! I guess I read the diff backwards. Periods are what we want, for sure.

**Pull request merged**